scripts/hera_mini_snap_data_catcher.py

```diff
@@ -212,7 +213,7 @@ def get_blt_index(antnum, time, ant_array, time_array):
                     )
                     if len(time_array) > 0:
                         file_len = TimeDelta(
-                            time_array[-1] - time_array[0], format="jd"
+                            (time_array[-1] - time_array[0]) * units.day, format="jd"
                         ).to_value("s")
```

Comment:
I don't actually see where this variable is used anywhere in the code. What am I missing?

Reply:
It's in the check in line 130 to know if it should write to disk or keep accumulating data.